libs/dh/admin/feature-user-management/src/lib/drawer/dh-user-drawer.component.html

```diff
@@ -16,7 +16,14 @@
 -->
 <ng-container *transloco="let t; read: 'admin.userManagement.drawer'">
   <watt-drawer #drawer size="large" (closed)="onClose()">
-    <watt-drawer-topbar></watt-drawer-topbar>
+    <watt-drawer-topbar>
+      <dh-user-status *ngIf="selectedUser" [status]="selectedUser.status"></dh-user-status>
+    </watt-drawer-topbar>
+    <watt-drawer-actions-content>
+      <div class="user-name__grid">
```

Comment:
suggestion: the extra element was necessary before, when there was a status badge under the heading, but the badge is not here anymore. The extra element and styling can be removed.